src/translations/en.json

```diff
@@ -0,0 +1,291 @@
+  },
+  "fields": {
+    "block": "block",
+    "blocks": "blocks",
+    "addLabel": "Add {{label}}",
+    "addNew": "Add new",
+    "addNewLabel": "Add new {{label}}",
+    "addRelationship": "Add relationship",
+    "blockType": "Block Type",
+    "chooseFromExisting": "Choose from existing",
+    "chooseLabel": "Choose {{label}}",
+    "collapseAll": "Collapse All",
+    "editLabelData": "Edit {{label}} data",
+    "itemsAndMore": "{{items}} and {{count}} more",
+    "labelRelationship": "{{label}} Relationship",
+    "latitude": "Latitude",
+    "linkedTo": "Linked to <0>label</0>",
+    "longitude": "Longitude",
+    "newLabel": "New {{label}}",
+    "passwordsDoNotMatch": "Passwords do not match.",
+    "relatedDocument": "Related Document",
+    "relationTo": "Relation To",
+    "removeUpload": "Remove Upload",
+    "saveChanges": "Save changes",
+    "searchForBlock": "Search for a block",
+    "selectExistingLabel": "Select existing {{label}}",
+    "showAll": "Show All",
+    "swapUpload": "Swap Upload",
+    "toggleBlock": "Toggle block",
+    "uploadNewLabel": "Upload new {{label}}"
+  },
+  "general": {
+    "aboutToDelete": "You are about to delete the {{label}} <1>{{title}}</1>. Are you sure?",
+    "addBelow": "Add Below",
+    "addFilter": "Add Filter",
+    "adminTheme": "Admin Theme",
+    "and": "And",
+    "automatic": "Automatic",
+    "backToDashboard": "Back to Dashboard",
+    "cancel": "Cancel",
+    "changesNotSaved": "Your changes have not been saved. If you leave now, you will lose your changes.",
+    "collections": "Collections",
+    "columnToSort": "Column to Sort",
+    "columns": "Columns",
+    "confirm": "Confirm",
+    "confirmDeletion": "Confirm deletion",
+    "confirmDuplication": "Confirm duplication",
+    "copied": "Copied",
+    "copy": "Copy",
+    "create": "Create",
+    "createNew": "Create New",
+    "createNewLabel": "Create new {{label}}",
+    "created": "Created",
+    "createdAt": "Created At",
+    "creating": "Creating",
+    "dark": "Dark",
+    "dashboard": "Dashboard",
+    "delete": "Delete",
+    "deletedSuccessfully": "Deleted successfully.",
+    "deleting": "Deleting...",
+    "descending": "Descending",
+    "duplicate": "Duplicate",
+    "duplicateWithoutSaving": "Duplicate without saving changes",
+    "editLabel": "Edit {{label}}",
+    "editing": "Editing",
+    "email": "Email",
+    "emailAddress": "Email Address",
+    "enterAValue": "Enter a value",
+    "filter": "Filter",
+    "filterWhere": "Filter {{label}} where",
+    "filters": "Filters",
+    "globals": "Globals",
+    "language": "Language",
+    "lastModified": "Last Modified",
+    "leaveAnyway": "Leave anyway",
+    "leaveWithoutSaving": "Leave without saving",
+    "light": "Light",
+    "loading": "Loading...",
+    "locales": "Locales",
+    "moveDown": "Move Down",
+    "moveUp": "Move Up",
+    "newPassword": "New Password",
+    "noFiltersSet": "No filters set",
+    "noLabel": "<No {{label}}>",
```

Comment:
This one comes from https://github.com/payloadcms/payload/blob/master/src/admin/components/views/collections/List/Cell/index.tsx#L39.

Maybe we have been inconsistent but I'm going to leave it for now since the other languages have it this way.